client/components/activity-card/types.d.ts

```diff
@@ -0,0 +1,14 @@
+// FUTURE WORK: universal typings location
```

Comment:
Out of curiosity, is there a particular reason why you chose `.d.ts` over `.ts`?

Reply:
I think I may have done this due to my own misunderstanding of what definition files in TypeScript are for. My thinking was that `*.d.ts` were good for enforcing that a file should only include type definitions, but reading further I believe now that this is mostly intended for public types, like if we were to publish a package using the code in this directory.

I could have probably gone with `types.ts` just as easily here, with no ill effects.